test/test_torch.py

```diff
@@ -13793,22 +13813,27 @@ def test_baddbmm(self, device, dtype):
         res2.baddbmm_(b1, b2)
         self.assertEqual(res2, res)
 
-        res2.baddbmm_(1, b1, b2)
+        self.assertWarnsRegex(
+            lambda: res2.baddbmm_(1, b1, b2),
+            "This signature for baddbmm_ is deprecated")
         self.assertEqual(res2, res * 2)
 
-        res2.baddbmm_(1, .5, b1, b2)
+        self.assertWarnsRegex(
+            lambda: res2.baddbmm_(1, .5, b1, b2),
+            "This signature for baddbmm_ is deprecated")
         self.assertEqual(res2, res * 2.5)
 
-        res3 = torch.baddbmm(1, res2, 0, b1, b2)
-        self.assertEqual(res3, res2)
+        self.assertWarnsRegex(
+            lambda: self.assertEqual(torch.baddbmm(1, res2, 0, b1, b2), res2),
+            "This signature for baddbmm is deprecated")
```

Comment:
I'm... a little perplexed by the rhyme and reason of the changes here? Did you just randomly pick some of these to use the deprecated form and some to not use the deprecated form?

Reply:
I kept 1 test with each form of the deprecated signature and moved the rest to the kwarg versions. Would you prefer if the tests using the deprecated signatures were also run with kwargs?

Reply:
I mean, I don't think there's anything wrong with what you did, but think of the developer in the future who is going to remove the deprecated signature and wants to know how to update these tests